diff --git a/utilities/lexicographicSortSchema.js b/utilities/lexicographicSortSchema.js
index ea884b68c75056071616f80f3901a25361699629..49ab981543dd88f3bed882a8e6be3ab44ea76e7a 100644
--- a/utilities/lexicographicSortSchema.js
+++ b/utilities/lexicographicSortSchema.js
@@ -97,6 +97,7 @@ function lexicographicSortSchema(schema) {
   function sortNamedType(type) {
     if (
       (0, _definition.isScalarType)(type) ||
+      (0, _definition.isEnumType)(type) ||
       (0, _introspection.isIntrospectionType)(type)
     ) {
       return type;
